uwuno/src/main/java/com/learning/uwuno/services/gameService.java

```diff
@@ -84,29 +83,10 @@ public void updateRoomStatus(String uid, String status) {
         }
     }
 
-    // TODO: Only has Lobby -> Start check. May need to add other states as necessary.
```

Comment:
Moved to utils since service class is going to only hold functions that can be directly mapped to a controller request.